apps/frontend/src/pages/ViewListing.tsx

```diff
@@ -31,6 +35,24 @@ interface ListingObject {
   images: { url: string }[];
   reviews?: Review[];
   status: string;
+  charityId: string;
+}
+
+interface CharityObject {
```

Comment:
I know this isn't normally the refactoring works, but maybe if we meet our 'deadline' early we could create a ticket for all of these re-used object types? I feel like creating a new file for 13 lines may be a little overkill especially since we've both already implemented this, but if we get a few more, I'd be down for that

Reply:
There's actually already a types.ts file in the frontend folder. But you could also use the vars defined in the Charities.tsx page I created. The variables are exported there. Not a biggie though. Feel free to resolve

Reply:
That's my bad, but I just realized that I can simplify my code quite a bit if I just specify the name as needed as a charity, I have added that object into the types.ts on my other PR and once it's merged we can reduce some of the replication.